src/main/java/com/google/cloud/pubsublite/flink/PubsubLiteSink.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.pubsublite.flink;
+
+import com.google.cloud.Tuple;
+import com.google.cloud.pubsublite.flink.sink.BulkWaitPublisher;
+import com.google.cloud.pubsublite.flink.sink.MessagePublisher;
+import com.google.cloud.pubsublite.flink.sink.PerServerPublisherCache;
+import com.google.cloud.pubsublite.flink.sink.SerializingPublisher;
+import com.google.errorprone.annotations.concurrent.GuardedBy;
+import java.time.Instant;
+import org.apache.flink.configuration.Configuration;
+import org.apache.flink.streaming.api.checkpoint.CheckpointedFunction;
+import org.apache.flink.streaming.api.functions.sink.RichSinkFunction;
+
+public class PubsubLiteSink<T> extends RichSinkFunction<T> implements CheckpointedFunction {
+  private final PubsubLiteSinkSettings<T> settings;
+
+  @GuardedBy("this")
+  private transient BulkWaitPublisher<Tuple<T, Instant>> publisher;
+
+  public PubsubLiteSink(PubsubLiteSinkSettings<T> settings) {
+    this.settings = settings;
+  }
+
+  @Override
+  public synchronized void snapshotState(
+      org.apache.flink.runtime.state.FunctionSnapshotContext functionSnapshotContext)
```

Comment:
Why not import these?

Reply:
Done. Intellij generated them like this and I never changed them